python/hail/matrixtable.py

```diff
@@ -1188,31 +1185,20 @@ def select_entries(self, *exprs: FieldRefArgs, **named_exprs: NamedExprs) -> 'Ma
         """
         exprs = [to_expr(e) if not isinstance(e, str) else self[e] for e in exprs]
         named_exprs = {k: to_expr(v) for k, v in named_exprs.items()}
-        strs = []
-        all_exprs = []
-        base, cleanup = self._process_joins(*itertools.chain(exprs, named_exprs.values()))
+        assignments = OrderedDict()
 
-        ids = []
         for e in exprs:
```

Comment:
It doesn't look like anything in `exprs` is getting passed to `_select_entries`. I might be missing something, but if not you should add a test that fails, and then fix `select_entries` to make it pass.